modules/stack/dependent_stacks.tf

```diff
@@ -0,0 +1,16 @@
+
+locals {
+  map_of_labels_array = {
+    for label in var.labels : split(":", label)[0] => split(":", label)[1]... # the ellipsis creates a group of values
+  }
+  depends_on_labels = local.map_of_labels_array["depends_on"]
```

Comment:
This local can be omitted.

Reply:
I think this is hooked up correctly now. The locals should all have refs now...